### PR TITLE
DO NOT MERGE Ana/deactivate polkadot fees for development purposes

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Only for development purposes 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
let's merge and not care about fees for now

**Comment by Bitcoinera** [comment]
> let's merge and not care about fees for now

👀

**Comment by Bitcoinera** [comment]
Maybe we can just deactivate them for ClaimRewards?

**Comment by faboweb** [comment]
ok it's a possibility. but even for sending it takes ages for me

**Comment by Bitcoinera** [comment]
> ok it's a possibility. but even for sending it takes ages for me

Best solution: we disable it for development

**Comment by faboweb** [comment]
ok

**Comment by faboweb** [comment]
but the UX sucks

**Comment by Bitcoinera** [comment]
> but the UX sucks

blame Polkadot xD

**Comment by Bitcoinera** [comment]
Done

**Comment by faboweb** [comment]
changelog missing

File: changes/ana_deactivate-polkadot-fees-for-development-purposes [hunk 1]
```diff
@@ -0,0 +1 @@
+[Changed] [#4001](https://github.com/cosmos/lunie/pull/4001) Disable Polkadot fees calculation in development @bitcoinera
```
Comment (Bitcoinera) [inline changes/ana_deactivate-polkadot-fees-for-development-purposes:1]: My handle is @Bitcoinera with uppercase 😛

**Pull request merged**